addons/hr_expense/models/hr_expense.py

```diff
@@ -275,6 +281,15 @@ def action_get_attachment_view(self):
         res['context'] = {'default_res_model': 'hr.expense', 'default_res_id': self.id}
         return res
 
+    @api.multi
+    def refuse_expense(self, reason):
+        self.ensure_one()
+        self.write({'is_refused': True})
+        self.sheet_id.write({'state': 'cancel'})
+        body = (_("Your Expense %s has been refused.<br/><ul class=o_timeline_tracking_value_list><li>Reason<span> : </span><span class=o_timeline_tracking_value>%s</span></li></ul>") % (self.name, reason))
```

Comment:
I think you could define a small qweb template. We try to remove most of inline html and use templates instead. Easier to customize.